### PR TITLE
Make augmentation_gallery works with ptyhon 2 and 3

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
Builds 37494.

File: docs/examples/augmentation_gallery.ipynb [hunk 1]
```diff
@@ -125,7 +126,7 @@
     "fig = plt.figure(figsize = (16,16))\n",
     "plt.suptitle(imagenet_synsets[pipe_out[0].at(n)[0]], fontsize=16)\n",
     "columns = 4\n",
-    "rows = len_outputs / columns + 1\n",
+    "rows = len_outputs // columns + 1\n",
```
[the remark this answers is not in the document]
Comment (JanuszL) [reply docs/examples/augmentation_gallery.ipynb]: In python 2 it doesn't make sense (as we got int anyway). I would stick to //.

**Comment by JanuszL** [comment]
Another build run 37496.

File: docs/examples/augmentation_gallery.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (JanuszL) [reply docs/examples/augmentation_gallery.ipynb]: Maybe I forgot to write it down, that we want this example to be runnable form python2 and python3 as well.

[the remark this answers is not in the document]
Comment (klecki) [reply docs/examples/augmentation_gallery.ipynb]: It was to not have additional row, this is currently floor(len_outputs / columns) + 1.

But it probably does not matter.

[the remark this answers is not in the document]
Comment (JanuszL) [reply docs/examples/augmentation_gallery.ipynb]: You are right, done.

**Pull request merged**